sdk/mobile-center-analytics/src/main/java/com/microsoft/azure/mobile/analytics/Analytics.java

```diff
@@ -130,11 +130,11 @@ public static void setEnabled(boolean enabled) {
     /**
      * Sets an analytics listener.
      *
-     * @param listener The custom crashes listener.
+     * @param listener The custom analytics listener.
      */
     @SuppressWarnings("WeakerAccess")
     @VisibleForTesting
-    protected static void setListener(AnalyticsListener listener) {
+    static void setListener(AnalyticsListener listener) {
```

Comment:
It needs to stay protected for Xamarin to change it back to public in bindings. Please turn it back to protected.